addons/website/views/snippets/s_rate.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_rate" name="Rate">
+    <section class="s_features_rate oe_img_bg pt48 pb48 show_description" style="background-image: url('/web/image/website.s_rate_default_image');">
+        <div class="container">
+            <div class="pb24" >
+                <h2>Menu</h2>
+                <h5 class="text-secondary">Add a menu description</h5>
+            </div>
+            <div class="row">
+                <div class="col-lg-6">
+                    <div class="pt24 pb16">
+                        <h3 class="text-primary">Starter</h3>
+                    </div>
+                    <div class="row">
+                        <div class="s_rate_dish col-lg-12 pt8 pb8" data-name="Box">
+                            <div class="d-flex align-items-baseline border-bottom mb8">
+                                <h4>Cheese Onion Rings</h4>
+                                <h5 class="ml-auto pl-3">$9.00</h5>
+                                <p class="s_rate_dish_description d-none"></p>
+                            </div>
+                            <p class="s_rate_dish_description">Add here a dish description</p>
```

Comment:
2 descriptions? ;) In fact put the s_rate_dish element in a sub-template that you can t-call everywhere to avoid code duplication and mistakes like these:
```xml
<template id="s_rate" name="Rate">
    <div ...>
        ...
        <div class="row">
            <t t-call="website.s_rate_dish">
                 <t t-set="name">...</t>
                 ....
            </t>
            ...
        </div>
    </div>
</template>

<template id="s_rate_dish">
    <div class="s_rate_dish col-lg-12 pt8 pb8" data-name="Box">
        <div class="d-flex align-items-baseline border-bottom mb8">
            <h4 t-esc="name"/>
            <h5 class="ml-auto pl-3" t-esc="price"/>
        </div>
       <p class="s_rate_dish_description">Add here a dish description</p>
    </div>
</template>
```
You can maybe use a better name than "Box" too